java/src/com/ibm/streamsx/topology/context/Placeable.java

```diff
@@ -0,0 +1,51 @@
+package com.ibm.streamsx.topology.context;
+
+import java.util.Set;
+
+import com.ibm.streamsx.topology.TopologyElement;
+
+/**
+ * Placement directives for a topology element when executing
+ * in a distributed runtime.
+ * <BR>
+ * Placement directives only apply when the topology
+ * is submitted to a {@link StreamsContext.Type#DISTRIBUTED DISTRIBUTED},
+ * {@link StreamsContext.Type#ANALYTICS_SERVICE ANALYTICS_SERVICE}
+ * or {@link StreamsContext.Type#DISTRIBUTED_TESTER DISTRIBUTED_TESTER} context.
+ * <BR>
+ * For all other context types directives are ignored.
+ */
+public interface Placeable<T extends Placeable<T>> extends TopologyElement {
+    
+    /**
+     * Fuse this container with other topology elements so that
+     * at runtime they all execute within the same operating system process.
```

Comment:
Since running in the same process is the current default behavior, would this override the default? As in, if you have these TStreams:
A->B->C->D

and you call

B.fuse(C)

Would it be the case that A and D run on one process, while B and C run on another?